### PR TITLE
Add/Update AWS Audit Manager > HIPAA Security Rule 2003 controls (2023-03-20). Closes #585

### LINKED ISSUE #585: Add/Update AWS Audit Manager > HIPAA Security Rule 2003 controls (2023-03-20)
**Is your feature request related to a problem? Please describe.**

**New Additions**

Benchmark | Control(s)
-- | --
164_308_a_1_ii_a | cloud_trail_enabled
  | config.1
164_308_a_1_ii_b | dynamodb_throughput_limit_check
  | ec2_instance_managed_by_ssm
  | ec2_managedinstance_association_compliance_status_check
  | ec2_managedinstance_patch_compliance_status_check
  | ec2_managedinstance_platform_check
  | efs_encrypted_check
  | elb_deletion_protection_enabled
  | encrypted_volumes
  | opensearch_encrypted_at_rest
  | opensearch_in_vpc_only
  | restricted_incoming_traffic
  | s3_account_level_public_access_blocks
  | s3_bucket_policy_grantee_check
164_308_a_1_ii_d | cloudfront_accesslogs_enabled
  | codebuild_project_logging_enabled
  | rds_logging_enabled
  | waf_classic_logging_enabled
  | wafv2_logging_enabled
164_308_a_3_i | opensearch_in_vpc_only
  | s3_account_level_public_access_blocks
  | s3_bucket_policy_grantee_check
164_308_a_3_ii_a | lambda_function_public_access_prohibited
  | rds_instance_iam_authentication_enabled
164_308_a_3_ii_b | s3_bucket_policy_grantee_check
164_308_a_4_i | rds_instance_iam_authentication_enabled
164_308_a_4_ii_a | efs_encrypted_check
  | elb_custom_security_policy_ssl_check
  | opensearch_encrypted_at_rest
  | waf_classic_logging_enabled
164_308_a_4_ii_b | s3_bucket_policy_grantee_check
164_308_a_4_ii_c | iam_password_policy
  | s3_bucket_policy_grantee_check
164_308_a_5_ii_b | ec2_managedinstance_applications_blacklisted
  | ec2_managedinstance_applications_required
164_308_a_5_ii_c | cloud_trail_cloud_watch_logs_enabled
  | cloud_trail_enabled
  | cloud_trail_log_file_validation_enabled
  | 3.6 -aws_security_hub
164_308_a_5_ii_d | iam_password_policy
  | secretsmanager_rotation_enabled_check
  | secretsmanager_scheduled_rotation_success_check
  | secretsmanager_secret_periodic_rotation
164_308_a_6_i | 3.3 - aws_security_hub
  | 3.6 - aws_security_hub
164_308_a_6_ii | cloud_trail_log_file_validation_enabled
  | 1.1 - aws_security_hub
  | 3.3 - aws_security_hub
  | 3.6 - aws_security_hub
164_308_a_7_i | backup_recovery_point_minimum_retention_check
164_308_a_7_ii_a | backup_recovery_point_minimum_retention_check
164_308_a_7_ii_b | backup_recovery_point_minimum_retention_check
164_308_a_7_ii_c | backup_recovery_point_minimum_retention_check
164_310_d_2_iv | aurora_resources_protected_by_backup_plan
  | backup_plan_min_frequency_and_min_retention_check
  | backup_recovery_point_encrypted
  | backup_recovery_point_manual_deletion_disabled
  | backup_recovery_point_minimum_retention_check
  | db_instance_backup_enabled
  | dynamodb_in_backup_plan
  | dynamodb_pitr_enabled
  | dynamodb_resources_protected_by_backup_plan
  | ebs_in_backup_plan
  | ebs_optimized_instance
  | ebs_resources_protected_by_backup_plan
  | ec2_resources_protected_by_backup_plan
  | efs_in_backup_plan
  | efs_resources_protected_by_backup_plan
  | elasticache_redis_cluster_automatic_backup_check
  | fsx_resources_protected_by_backup_plan
  | rds_in_backup_plan
  | rds_resources_protected_by_backup_plan
  | redshift_backup_enabled
  | s3_bucket_replication_enabled
  | s3_bucket_versioning_enabled
164_312_a_1 | opensearch_in_vpc_only
  | s3_bucket_policy_grantee_check
164_312_a_2_ii | backup_recovery_point_minimum_retention_check
164_312_a_2_iv | efs_encrypted_check
  | encrypted_volumes
  | opensearch_encrypted_at_rest
  | kms.1 - aws_security_hub
  | kms.2 - aws_security_hub
164_312_b | waf_classic_logging_enabled
164_312_c_1 | dax_encryption_enabled
  | dynamodb_table_encryption_enabled
  | ec2_ebs_encryption_by_default
  | eks_secrets_encrypted
  | elasticsearch_encrypted_at_rest
  | elasticsearch_node_to_node_encryption_check
  | elb_tls_https_listeners_only
  | opensearch_encrypted_at_rest
  | opensearch_https_required
  | opensearch_node_to_node_encryption_check
  | rds_snapshot_encrypted
  | rds_storage_encrypted
  | redshift_cluster_kms_enabled
  | redshift_require_tls_ssl
  | s3_default_encryption_kms
  | ec2.3 - aws_security_hub
164_312_c_2 | ec2.3 - aws_security_hub
164_312_d | iam_password_policy
164_312_e_1 | elb_custom_security_policy_ssl_check
  | opensearch_node_to_node_encryption_check
164_312_e_2_ii | efs_encrypted_check
  | encrypted_volumes
  | opensearch_encrypted_at_rest
164_314_b_1 | cloud_trail_encryption_enabled
  | dax_encryption_enabled
  | dynamodb_table_encryption_enabled
  | ec2_ebs_encryption_by_default
  | eks_secrets_encrypted
  | elasticsearch_encrypted_at_rest
  | elasticsearch_node_to_node_encryption_check
  | elb_tls_https_listeners_only
  | encrypted_volumes
  | opensearch_encrypted_at_rest
  | opensearch_https_required
  | opensearch_node_to_node_encryption_check
  | rds_snapshot_encrypted
  | rds_storage_encrypted
  | redshift_cluster_kms_enabled
  | redshift_cluster_public_access_check
  | redshift_require_tls_ssl
  | s3_account_level_public_access_blocks
  | s3_bucket_public_read_prohibited
  | s3_bucket_public_write_prohibited
  | s3_bucket_server_side_encryption_enabled
  | s3_bucket_ssl_requests_only
  | s3_default_encryption_kms
  | sagemaker_endpoint_configuration_kms_key_configured
  | sagemaker_notebook_instance_kms_key_configured
  | service_vpc_endpoint_enabled
164_314_b_2 | cloud_trail_encryption_enabled
  | dax_encryption_enabled
  | dynamodb_table_encryption_enabled
  | ec2_ebs_encryption_by_default
  | eks_secrets_encrypted
  | elasticsearch_encrypted_at_rest
  | elasticsearch_node_to_node_encryption_check
  | elb_tls_https_listeners_only
  | encrypted_volumes
  | opensearch_encrypted_at_rest
  | opensearch_https_required
  | opensearch_node_to_node_encryption_check
  | rds_snapshot_encrypted
  | rds_storage_encrypted
  | redshift_cluster_kms_enabled
  | redshift_cluster_public_access_check
  | redshift_require_tls_ssl
  | s3_account_level_public_access_blocks
  | s3_bucket_public_read_prohibited
  | s3_bucket_public_write_prohibited
  | s3_bucket_server_side_encryption_enabled
  | s3_bucket_ssl_requests_only
  | s3_default_encryption_kms
  | sagemaker_endpoint_configuration_kms_key_configured
  | sagemaker_notebook_instance_kms_key_configured
  | service_vpc_endpoint_enabled

**Removed Controls**

Benchmark | Control (s)
-- | --
164_308_a_1_ii_a | config_enabled_all_regions
164_308_a_1_ii_b | ebs_volume_encryption_at_rest_enabled
  | efs_file_system_encrypt_data_at_rest
  | elb_application_lb_deletion_protection_enabled
  | s3_public_access_block_account
  | vpc_security_group_restrict_ingress_common_ports_all
164_308_a_3_i | s3_public_access_block_account
164_308_a_3_ii_a | apigateway_stage_logging_enabled
164_308_a_3_ii_a | vpc_flow_logs_enabled
164_308_a_4_ii_a | ebs_volume_encryption_at_rest_enabled
  | efs_file_system_encrypt_data_at_rest
164_308_a_4_ii_c | iam_account_password_policy_strong_min_reuse_24
164_308_a_5_ii_c | log_metric_filter_console_authentication_failure
  | securityhub_enabled
164_308_a_5_ii_d | iam_account_password_policy_min_length_14
  | iam_account_password_policy_one_lowercase_letter
  | iam_account_password_policy_one_number
  | iam_account_password_policy_one_symbol
  | iam_account_password_policy_one_uppercase_letter
  | iam_account_password_policy_reuse_24
  | iam_password_policy_expire_90
164_308_a_6_i | log_metric_filter_console_authentication_failure
  | log_metric_filter_root_login
164_308_a_6_ii | log_metric_filter_console_authentication_failure
  | log_metric_filter_root_login
164_312_a_2_iv | ebs_volume_encryption_at_rest_enabled
  | efs_file_system_encrypt_data_at_rest
  | kms_cmk_rotation_enabled
  | kms_key_decryption_restricted_in_iam_customer_managed_policy
  | kms_key_decryption_restricted_in_iam_inline_policy
  | kms_key_not_pending_deletion
  | redshift_cluster_encryption_in_transit_enabled
  | s3_bucket_enforces_ssl
164_312_b | cloudtrail_s3_data_events_enabled
  | redshift_cluster_encryption_logging_enabled
  | s3_bucket_logging_enabled
  | securityhub_enabled
164_312_c_1 | s3_bucket_object_lock_enabled
164_312_c_2 | ebs_attached_volume_encryption_enabled
  | s3_bucket_object_lock_enabled
164_312_d | iam_account_password_policy_strong_min_reuse_24
164_312_e_1 | ec2_instance_in_vpc
  | es_domain_in_vpc
  | lambda_function_in_vpc
  | s3_bucket_enforces_ssl
  | vpc_security_group_restrict_ingress_common_ports_all
  | vpc_security_group_restrict_ingress_ssh_all
  | vpc_security_group_restrict_ingress_tcp_udp_all
164_312_e_2_ii | ebs_volume_encryption_at_rest_enabled
  | efs_file_system_encrypt_data_at_rest
  | elb_application_lb_redirect_http_request_to_https
  | elb_classic_lb_use_ssl_certificate
  | redshift_cluster_encryption_in_transit_enabled
  | s3_bucket_enforces_ssl

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by madhushreeray30**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: hipaa_security_rule_2003/164_314/164_314.sp [hunk 1]
```diff
@@ -0,0 +1,17 @@
+locals {
+  hipaa_security_rule_2003_164_314_common_tags = merge(local.hipaa_security_rule_2003_common_tags, {
+    hipaa_section = "164_314"
+  })
+}
+
+benchmark "hipaa_security_rule_2003_164_314" {
+  title       = "164.314 Technical Safeguards"
```
Comment (khushboo9024) [inline hipaa_security_rule_2003/164_314/164_314.sp:8]:
```suggestion
  title       = "164.314 Organizational Requirements"
```

File: hipaa_security_rule_2003/164_314/164_314.sp [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  description = "The Security Rule defines technical safeguards in § 164.304 as `the technology and the policy and procedures for its use that protect electronic protected health information and control access to it.`"
```
Comment (khushboo9024) [inline hipaa_security_rule_2003/164_314/164_314.sp:9]: Description is not correct .

File: conformance_pack/cloudwatch.sp [hunk 3]
```diff
@@ -91,9 +91,8 @@ control "log_metric_filter_console_login_mfa" {
   query       = query.log_metric_filter_console_login_mfa
 
   tags = merge(local.conformance_pack_cloudwatch_common_tags, {
-    gdpr     = "true"
-    hipaa    = "true"
-    nist_csf = "true"
+    gdpr                     = "true"
+    nist_csf                 = "true"
```
Comment (khushboo9024) [inline conformance_pack/cloudwatch.sp:95]: Please fix indentation .

File: conformance_pack/opensearch.sp [hunk 4]
```diff
@@ -80,3 +122,182 @@ query "opensearch_domain_in_vpc" {
...
+      arn as resource,
+      case
+        when encryption_at_rest_options ->> 'Enabled' = 'false' then 'alarm'
+        else 'ok'
+      end status,
+      case
+        when encryption_at_rest_options ->> 'Enabled' = 'false' then title || ' encryption at rest not enabled.'
+        else title || ' encryption at rest enabled.'
+      end reason
+
```
Comment (khushboo9024) [inline conformance_pack/opensearch.sp:138]:
```suggestion
```

File: conformance_pack/opensearch.sp [hunk 5]
```diff
@@ -80,3 +122,182 @@ query "opensearch_domain_in_vpc" {
...
+query "opensearch_domain_encryption_at_rest_enabled" {
+  sql = <<-EOQ
+    select
+      arn as resource,
+      case
+        when encryption_at_rest_options ->> 'Enabled' = 'false' then 'alarm'
+        else 'ok'
+      end status,
+      case
+        when encryption_at_rest_options ->> 'Enabled' = 'false' then title || ' encryption at rest not enabled.'
```
Comment (khushboo9024) [inline conformance_pack/opensearch.sp:135]:
```suggestion
        when encryption_at_rest_options ->> 'Enabled' = 'false' then title || ' encryption at rest disabled.'
```

File: conformance_pack/secretsmanager.sp [hunk 6]
```diff
@@ -24,10 +24,11 @@ control "secretsmanager_secret_rotated_as_scheduled" {
...
   tags = merge(local.conformance_pack_secretsmanager_common_tags, {
-    gxp_21_cfr_part_11 = "true"
-    nist_800_53_rev_4  = "true"
-    nist_800_53_rev_5  = "true"
-    nist_csf           = "true"
+    gxp_21_cfr_part_11       = "true"
+    nist_800_53_rev_4        = "true"
+    nist_800_53_rev_5        = "true"
+    nist_csf                 = "true"
+    hipaa_security_rule_2003 = "true"
```
Comment (khushboo9024) [inline conformance_pack/secretsmanager.sp:31]: Please sort the tags.

File: conformance_pack/vpc.sp [hunk 7]
```diff
@@ -939,6 +950,38 @@ query "vpc_security_group_restrict_ingress_kafka_port" {
...
+      select
+        group_id,
+        count(*)
+      from
+        aws_vpc_security_group_rule
+      where
+        type = 'ingress'
+        and cidr_ipv4 = '0.0.0.0/0'
+        and (from_port is null or from_port not in (80,443))
+      group by group_id
```
Comment (khushboo9024) [inline conformance_pack/vpc.sp:965]:
```suggestion
      group by 
         group_id
```

File: conformance_pack/waf.sp [hunk 8]
```diff
@@ -1,6 +1,76 @@
-# Non-Config rule query
+locals {
+  conformance_pack_waf_common_tags = merge(local.aws_compliance_common_tags, {
+    service = "AWS/waf"
```
Comment (khushboo9024) [inline conformance_pack/waf.sp:3]:
```suggestion
    service = "AWS/WAF
```

File: conformance_pack/waf.sp [hunk 9]
```diff
@@ -1,6 +1,76 @@
...
+  query       = query.waf_regional_rule_condition_attached
+
+  tags = merge(local.conformance_pack_waf_common_tags, {
+    audit_manager_pci_v321 = "true"
+    well_architected       = "true"
+  })
+}
+
+control "waf_rule_group_rule_attached" {
+  title       = "A WAF global rule group should have at least one rule"
```
Comment (khushboo9024) [inline conformance_pack/waf.sp:19]:
```suggestion
  title       = "WAF global rule group should have at least one rule"
```

File: conformance_pack/waf.sp [hunk 10]
```diff
@@ -1,6 +1,76 @@
...
+  query       = query.waf_rule_group_rule_attached
+
+  tags = merge(local.conformance_pack_waf_common_tags, {
+    audit_manager_pci_v321 = "true"
+    well_architected       = "true"
+  })
+}
+
+control "waf_web_acl_rule_attached" {
+  title       = "A WAF global web ACL should have at least one rule or rule group"
```
Comment (khushboo9024) [inline conformance_pack/waf.sp:30]:
```suggestion
  title       = "WAF global web ACL should have at least one rule or rule group"
```

File: conformance_pack/waf.sp [hunk 11]
```diff
@@ -56,3 +125,82 @@ query "waf_web_acl_rule_attached" {
...
+        when rules is null or jsonb_array_length(rules) = 0 then 'alarm'
+        else 'ok'
+      end as status,
+      case
+        when rules is null or jsonb_array_length(rules) = 0 then title || ' has no attached rules.'
+        else title || ' has attached rules.'
+      end as reason,
+      -- Additional Dimensions
+      region,
+      account_id
```
Comment (khushboo9024) [inline conformance_pack/waf.sp:162]:
```suggestion
      end as reason
      ${local.tag_dimensions_sql}
      ${local.common_dimensions_sql}
```

File: conformance_pack/wafv2.sp [hunk 12]
```diff
@@ -10,20 +10,30 @@ control "wafv2_web_acl_logging_enabled" {
...
+    nist_800_53_rev_4        = "true"
+    nist_800_53_rev_5        = "true"
+    nist_csf                 = "true"
+    rbi_cyber_security       = "true"
+    soc_2                    = "true"
+  })
+}
+
+control "waf_web_acl_logging_enabled" {
+  title       = "Logging should be enabled on AWS WAF regional and global web access control list (ACLs)"
```
Comment (khushboo9024) [inline conformance_pack/wafv2.sp:31]:
```suggestion
  title       = "WAF web ACL logging should be enabled"
```

File: hipaa_security_rule_2003/164_314/164_314_b_1.sp [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+    control.s3_bucket_enforces_ssl,
+    control.s3_bucket_restrict_public_read_access,
+    control.s3_bucket_restrict_public_write_access,
+    control.s3_public_access_block_account,
+    control.sagemaker_endpoint_configuration_encryption_at_rest_enabled,
+    control.sagemaker_notebook_instance_encryption_at_rest_enabled,
+    control.vpc_endpoint_service_acceptance_required_enabled
+  ]
+
+  tags = merge(local.hipaa_security_rule_2003_164_312_common_tags, {
```
Comment (khushboo9024) [inline hipaa_security_rule_2003/164_314/164_314_b_1.sp:33]:
```suggestion
  tags = merge(local. hipaa_security_rule_2003_164_314_common_tags, {
```

File: hipaa_security_rule_2003/164_314/164_314_b_2.sp [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+    control.s3_bucket_enforces_ssl,
+    control.s3_bucket_restrict_public_read_access,
+    control.s3_bucket_restrict_public_write_access,
+    control.s3_public_access_block_account,
+    control.sagemaker_endpoint_configuration_encryption_at_rest_enabled,
+    control.sagemaker_notebook_instance_encryption_at_rest_enabled,
+    control.vpc_endpoint_service_acceptance_required_enabled
+  ]
+
+  tags = merge(local.hipaa_security_rule_2003_164_312_common_tags, {
```
Comment (khushboo9024) [inline hipaa_security_rule_2003/164_314/164_314_b_2.sp:33]:
```suggestion
  tags = merge(local. hipaa_security_rule_2003_164_314_common_tags, {
```

**Review by rajlearner17 — APPROVED** [review]
LGTM

File: conformance_pack/rds.sp [hunk 15]
```diff
@@ -205,15 +217,15 @@ control "rds_db_instance_deletion_protection_enabled" {
...
-    nist_800_53_rev_4      = "true"
-    nist_800_53_rev_5      = "true"
-    soc_2                  = "true"
+    cisa_cyber_essentials                  = "true"
+    fedramp_low_rev_4                      = "true"
+    fedramp_moderate_rev_4                 = "true"
+    ffiec                                  = "true"
+    gxp_21_cfr_part_11                     = "true"
+    hipaa_final_omnibus_security_rule_2013 = "true"
+    nist_800_53_rev_4                      = "true"
```
Comment (misraved) [inline conformance_pack/rds.sp:226]: I see we have removed `nist_800_171_rev_2` tag, is that correct?

File: conformance_pack/vpc.sp [hunk 16]
```diff
@@ -300,11 +310,12 @@ control "vpc_network_acl_unused" {
 
 control "vpc_security_group_allows_ingress_authorized_ports" {
   title       = "VPC Security groups should only allow unrestricted incoming traffic for authorized ports"
-  description = "This control checks whether the VPC security groups that are in use allow unrestricted incoming traffic. Optionally the rule checks whether the port numbers are listed in the authorizedTcpPorts parameter. The default values for authorizedTcpPorts are 80 and 443."
+  description = "This control checks whether the security groups that are in use allow unrestricted incoming traffic. Optionally the rule checks whether the port numbers are listed in the authorizedTcpPorts parameter. The default values for authorizedTcpPorts are 80 and 443."
```
Comment (misraved) [inline conformance_pack/vpc.sp:313]: Why was the description updated?

File: conformance_pack/waf.sp [hunk 17]
```diff
@@ -140,40 +141,40 @@ query "waf_web_acl_logging_enabled" {
...
       case
-        when predicates is null or jsonb_array_length(predicates) = 0 then 'alarm'
-        else 'ok'
+        when jsonb_array_length(resources) > 0 then 'ok'
+        else 'alarm'
       end as status,
       case
-        when predicates is null or jsonb_array_length(predicates) = 0 then title || ' has no attached conditions.'
-        else title || ' has attached conditions.'
+        when jsonb_array_length(resources) > 0 then title || ' assoicated with any AWS resource.'
```
Comment (misraved) [inline conformance_pack/waf.sp:153]:
```suggestion
        when jsonb_array_length(resources) > 0 then title || ' associated with AWS resource(s).'
```

File: hipaa_final_omnibus_security_rule_2013/164_308/164_308.sp [hunk 18]
```diff
@@ -0,0 +1,37 @@
+locals {
+  hipaa_final_omnibus_security_rule_2013_164_308_common_tags = merge(local.hipaa_final_omnibus_security_rule_2013_common_tags, {
+    hipaa_section = "164_308"
+  })
+}
+
+benchmark "hipaa_final_omnibus_security_rule_2013_164_308" {
+  title       = "164.308 Administrative Safeguards"
+  description = "An important step in strengthen the protection of protected health information, especially in electronic form, as well as give patients more access to their individual health information. The HIPAA Omnibus Rule is a set of final regulations that modifies the existing HIPAA rules and implements a variety of provisions of the Health Information Technology for Economic and Clinical Health (HITECH) Act."
```
Comment (misraved) [inline hipaa_final_omnibus_security_rule_2013/164_308/164_308.sp:9]:
```suggestion
  description = "An important step in strengthening the protection of health information, especially in electronic form, as well as give patients more access to their individual health information. The HIPAA Omnibus Rule is a set of final regulations that modifies the existing HIPAA rules and implements a variety of provisions of the Health Information Technology for Economic and Clinical Health (HITECH) Act."
```
Is this the correct benchmark description?

File: hipaa_final_omnibus_security_rule_2013/164_314/164_314_b_2.sp [hunk 19]
```diff
@@ -0,0 +1,36 @@
+benchmark "hipaa_final_omnibus_security_rule_2013_164_314_b_2" {
+  title       = "164.314(b)(2) Implementation specifications"
+  description = "The plan documents of the group health plan must be amended to incorporate provisions to require the plan sponsor to?"
```
Comment (misraved) [inline hipaa_final_omnibus_security_rule_2013/164_314/164_314_b_2.sp:3]: Is the description correct? Should we have a `?` at the end of the description?

File: hipaa_security_rule_2003/164_308/164_308_a_4_ii_a.sp [hunk 20]
```diff
@@ -1,4 +1,4 @@
-benchmark "hipaa_164_308_a_4_ii_a" {
+benchmark "hipaa_security_rule_2003_164_308_a_4_ii_a" {
   title       = "164.308(a)(4)(ii)(A) Isolating health care clearinghouse functions"
```
Comment (misraved) [inline hipaa_security_rule_2003/164_308/164_308_a_4_ii_a.sp:2]:
```suggestion
  title       = "164.308(a)(4)(ii)(A) Isolating healthcare clearing house functions"
```
Is the title correct?

**Review by misraved — COMMENTED** [review]
Please take a look at the minor review comments. Thanks!!

File: conformance_pack/glue.sp [hunk 21]
```diff
@@ -17,7 +17,7 @@ control "glue_dev_endpoint_cloudwatch_logs_encryption_enabled" {
 control "glue_dev_endpoint_job_bookmarks_encryption_enabled" {
   title       = "Glue dev endpoints job bookmark encryption should be enabled"
   description = "Ensure Glue dev endpoints have job bookmark encryption enabled to protect sensitive information at rest."
-  query       = query.glue_dev_endpoint_job_bookmark_encryption_enabled
+  query       = query.glue_dev_endpoint_job_bookmarks_encryption_enabled
```
[the remark this answers is not in the document]
Comment (khushboo9024) [reply conformance_pack/glue.sp:20]: This was changed as we generally keep query and control name same.

**Review by misraved — APPROVED** [review]

**Comment by misraved** [comment]
@khushboo9024 could you please fix the ordering of `HIPAA Security Rule 2003` and `HIPAA Final Omnibus Security Rule 2013` benchmark references in docs?

**Pull request merged**